app/controllers/discover.js

```diff
@@ -383,6 +387,10 @@ export default Ember.Controller.extend(Analytics, {
                 });
         },
 
+        clearQueryString() {
```

Comment:
I don't see where this is being called, it looks like it's not needed. Is this actually needed?